lib/suspenders/generators/app_generator.rb

```diff
@@ -14,6 +14,9 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :heroku_flags, type: :string, default: "",
       desc: "Set extra Heroku flags"
 
+    class_option :ci, type: :string, default: "circle",
+      desc: "Specify a CI provider (options: circle, travis, codeship)"
```

Comment:
Is there already a system in place to validate this input? Where would be the best place to validate that?